include/multipass/sshfs_mount/sshfs_mount.h

```diff
@@ -29,6 +29,12 @@ class SftpServer;
 class SshfsMount
 {
 public:
+    enum class SshfsPkgType
+    {
+        debian,
+        snap
+    };
+
```

Comment:
I would quite like to not have this enum…

Reply:
Yeah, me neither :grin:  I had this mainly to quirk the options based on what type of `libfuse` package is installed.  Maybe probing `libfuse` for the version or something would be better.

Reply:
I got rid of this.

Reply:
Did you though?